main.py

```diff
@@ -1,15 +1,29 @@
 import datetime
+import os
+import time
 
-def obter_limite():
-    cargo_atual = str(input("Para fazermos sua analise de credito, "
-                            "precisamos que voce entre com algumas informacoes:\nCargo atual: "))
-    salario_atual = float(input("Salario Atual: "))
-    ano_de_nascimento = int(input("Ano de nascimento: "))
-    ano_vigente = datetime.date.today()
-    idade_aproximada = (ano_vigente.year - ano_de_nascimento)
+
+NOME_COMPLETO = "Rodiney Branta"
+
+
+def obter_limite(salario_atual, idade_aproximada):
     valor_credito_disponivel = ((salario_atual*(idade_aproximada / 1000)) +100) 
-    print(f"Seu cargo atual: {cargo_atual} \nSeu Salario atual: {salario_atual} \nSeu ano de nascimento: {ano_de_nascimento}")
-    print(f"Voce tem aproximadamente {idade_aproximada} anos.")
-    print(f"Voce possui o valor em credito de R$ {valor_credito_disponivel:.2f}, disponiveis para compras em nossa loja.")
+    return valor_credito_disponivel
+
+print(f"Seja bem-vindo a Poppy loja , sou {NOME_COMPLETO}, entre e escolha um produto")
+
+cargo_atual = str(input("Para fazermos sua analise de credito, "
+                        "solicitamos que voce entre com algumas informacoes:\nCargo atual: "))
```

Comment:
Sempre cuide para isolar as responsabilidades no código fonte. Há momentos em que parece preciosismo, mas acredite que no _long run_, a gente sempre sai ganhando com isto.

Aqui eu não vejo porque fixar a mensagem de boas-vindas com o pedido do cargo da pessoa. Se você precisar em algum momento adicionar uma pergunta antes, vai ter de mover a mensagem de boas-vindas para esta nova pergunta, ou pode ocorrer de esquecer e ela aparecer onde não deveria.

```suggestion
print(f"Para fazermos sua análise de crédito, solicitamos que você entre com algumas informações:")
cargo_atual = str(input("Cargo atual: "))
```

Reply:
> Sempre cuide para isolar as responsabilidades no código fonte. Há momentos em que parece preciosismo, mas acredite que no _long run_, a gente sempre sai ganhando com isto.
> 
> Aqui eu não vejo porque fixar a mensagem de boas-vindas com o pedido do cargo da pessoa. Se você precisar em algum momento adicionar uma pergunta antes, vai ter de mover a mensagem de boas-vindas para esta nova pergunta, ou pode ocorrer de esquecer e ela aparecer onde não deveria.

Estou estudando uma forma, de mover a mensagem de boas-vindas e modularizar o while, para que eu possa utilizar _if __name__ == '__função__':_  para controlar a chamada das funções do modulo modularizando.py.